terra2/assetlist.json

```diff
@@ -1643,4 +1643,57 @@
       }
     }
   ]
-}
\ No newline at end of file
+},
+{
```

Comment:
This asset needs to go inside the `assets` array

Reply:
@JeremyParish69 Corrected